fearless/ru.lproj/Localizable.strings

```diff
@@ -692,7 +704,39 @@ and depending on the network state, may not be selected to participate";
+"staking.start.change.collators.suggested.subtitle" = "FEARLESS algorithm has made a list of suggested collators based on the following criteria:";
+"staking.select.suggested" = "Select suggested";
+"staking.start.change.collators.custom.title" = "Stake with your collators";
+"select.collators.warning" = "WARNING: Holders of MOVR/GLMR tokens should perform careful due diligence on collators before delegating. Being listed as a collator is not an endorsement or recommendation from the Moonbeam Network, the Moonriver Network, or Moonbeam Foundation. Neither the Moonbeam Network, Moonriver Network, nor Moonbeam Foundation has vetted the list collators and assumes no responsibility with regard to the selection, performance, security, accuracy, or use of any third-party offerings. You alone are responsible for doing your own diligence to understand the applicable fees and all risks present, including actively monitoring the activity of your collators.";
+"staking.suggested.collators.title" = "Suggested collators";
 "wallet.send.tip.title" = "Чаевые";
+"staking.selected.collator" = "Выбранный коллатор";
+"parachain.staking.minimum.bond" = "Min. bond";
+"parachain.staking.self.bonded" = "Self-bonded";
+"parachain.staking.effective.amount.bonded" = "Effective amount bonded";
+"parachain.staking.ready.for.revoking" = "Ready for unlocking";
+"parachain.staking.stake.less" = "Stake less";
+"parachain.staking.revoke" = "Revoke";
+"parachain.staking.collator" = "Collator";
+"parachain.staking.unlock" = "Unlock";
+"parachain.staking.filters.effective.amount.bonded" = "Effective amount bonded (%@)";
+"parachain.staking.filters.collator.own.stake" = "Collator's own stake (%@)";
+"parachain.staking.filters.delegations" = "Delegations";
+"parachain.staking.filters.minimum.bond" = "Minimum bond";
+"staking.your.collator.title" = "Ваш коллатор";
+"custom.collators.title" = "Stake with known collators";
+"custom.collators.text" = "You should trust your collators to act competently and honest; basing your decision purely on their current profitability could lead to reduced profits or even loss of funds.";
+"staking.next.round" = "NEXT ROUND";
 "chain.selection.all.networks" = "Все сети";
 "chain.selection.hide.zero.balances" = "Скрыть с пустым балансом";
+"staking.stake.with.selected.title" = "Stake with selected";
+"staking.filter.title.rewards.apr" = "Rewards (APR)";
+"staking.custom.collators.title" = "Select a collator";
+"parachain.staking.recommended.section.title" = "Collators suggested by algorithm";
+"staking.recommended.filter.minimum.bond" = "Having relevant minimum bond";
+"staking.recommended.collators.empty.text" = "Your stake amount is less than the minimum one of all collators suggested by the algorithm. Increase your stake amount or choose a collator manually";
+"parachain.staking.reward.info.max" = "Максимальный APR";
+"parachain.staking.reward.info.avg" = "Средний APR";
+"parachain.staking.recommended.list.title" = "Предложено %d";
```

Comment:
file with English text

Reply:
it's okay, that means there is no translation in lokalise. we doesn't have 100% russian translation